index.html

```diff
@@ -25,7 +25,18 @@
         <h1>Hey There. <br> I’m Ayodele</h1>
         <h3>I’am a Software Developer</h3>
         <p>I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
-        <div class="item"><img src="img_social.png" alt="icons"></div>
+        
+        <ul class="icongroup">
+            <li class="icon-item githubx"><a href="#"><img src="pictures/git_white.svg" alt="Github"></a></li>
+    
+            <li class="icon-item linkedinx"><a href="#"><img  src="pictures/instagram_white.svg" alt="linkedin"></a></li>
+    
+            <li class="icon-item angelistx"><a href="#"><img src="pictures/angelist_white.svg" alt="angel.co"></a></li>
+    
+            <li class="icon-item twitterx"><a href="#"><img src="pictures/twitter_white.svg" alt="twitter"></a></li>
+    
+            <li class="icon-item mediumx"><a href="#"><img src="pictures/medium_white.svg" alt="medium.com"></a></li>
+        </ul>
```

Comment:
### [OPTIONAL]
- [ ]  Instead of using fixed with on your section as well as on your header, I would recommend you kindly use the responsive width(percentages), just go on your CSS and change the fixed width:375px with width:100% only on the section below :point_down: :point_down: 
![Screenshot from 2021-12-10 12-40-07](https://user-images.githubusercontent.com/65068771/145562640-fb0c7f74-93e3-4a9c-afdd-fc042bb70fce.png)

#### Note that this will adapt your mobile version on different mobile screen

- [ ] You can also improve the styling when the input is focused, just like
### [OPTIONAL]
- [ ]  Instead of using fixed with on your section as well as on your header, I would recommend you kindly use the responsive width(percentages), just go on your CSS and change the fixed width:375px with width:100% only on the section below :point_down: :point_down: 
![Screenshot from 2021-12-10 12-40-07](https://user-images.githubusercontent.com/65068771/145562640-fb0c7f74-93e3-4a9c-afdd-fc042bb70fce.png)

#### Note that this will adapt your mobile version on different mobile screen